doc/developer-guide/developer-guide.rst

```diff
@@ -3,8 +3,8 @@ TunableLaser Developer Guide
 #########################################
 
 The CSC is a standard salobj CSC.
-The TunableLaser's communication with the hardware is written in Python using the PySerial library.
-The laser is connected to a Raspberry Pi using an RS-232 to USB converter cable.
+The TunableLaser's communication with the hardware is written in Python using the async TCP/IP library.
+The laser is connected to a serial terminal.
```

Comment:
I suggest the term "terminal server" (here and elsewhere). It's the usual terminology and is a term one can google.